data/World/Forest Temple.json

```diff
@@ -124,7 +148,9 @@
         "region_name": "Forest Temple Outside Upper Ledge",
         "dungeon": "Forest Temple",
         "locations": {
-            "Forest Temple Floormaster Chest": "True"
+            "Forest Temple Floormaster Chest": "True",
+            "Forest Temple Courtyard Recovery Heart 1": "True",
+            "Forest Temple Courtyard Recovery Heart 2": "True"
```

Comment:
These can be obtained from below with a Boomerang trick throw. I'm gonna keep pointing out tricks where I see them but honestly don't worry about tricks, I'm mainly leaving these as notes to myself for later.